tools/cli/commands.js

```diff
@@ -925,7 +925,8 @@ on an OS X system.");
       //     packages with binary npm dependencies
       serverArch: bundleArch,
       buildMode: options.debug ? 'development' : 'production',
-    }
+    },
+    providePackageJSONForUnavailableBinaryDeps: !!process.env.METEOR_BINARY_DEP_WORKAROUND,
```

Comment:
OK sgtm. I wish we had some place where all of the special env vars were documented.